doc/lua_api.txt

```diff
@@ -4938,18 +4938,15 @@ This is basically a reference to a C++ `ServerActiveObject`
     * element `stat` values:
       `position`, `name`, `scale`, `text`, `number`, `item`, `dir`
 * `hud_get(id)`: gets the HUD element definition structure of the specified ID
-* `hud_set_flags(flags)`: sets specified HUD flags to `true`/`false`
-    * `flags`: (is visible) `hotbar`, `healthbar`, `crosshair`, `wielditem`,
-      `breathbar`, `minimap`, `minimap_radar`
-    * pass a table containing a `true`/`false` value of each flag to be set or
-      unset.
+* `hud_set_flags(flags)`: sets specified HUD flags of player.
+    * A table containing a `true`/`false` value of the HUD flags to be set or
+      unset. See `Player HUD Flags` section for a list of possible flags.
```

Comment:
I actually think it's best to list the table entries here instead of elsewhere (which is annoying to find).

Reply:
Agreed. This table is only relevant to `hud_get_flags` and `hud_set_flags`, which are right next to each other...